.github/workflows/fast-forward-branch.yml

```diff
@@ -22,11 +22,11 @@ jobs:
       - id: git_log
         name: Generate diff log
         run: |
-          issues=$(git log origin/${{ inputs.to_branch }}..origin/${{ inputs.ref }} | grep -ioh "THREESCALE-[0-9]\+" | sort -u | sed -e 's/^/https:\/\/issues.redhat.com\/browse\//')
+          issues=$(git log origin/${{ inputs.to_branch }}..${{ inputs.ref }} | grep -ioh "THREESCALE-[0-9]\+" | sort -u | sed -e 's/^/https:\/\/issues.redhat.com\/browse\//')
           echo "issues<<EOF" >> $GITHUB_OUTPUT
           echo "$issues" >> $GITHUB_OUTPUT
           echo "EOF" >> $GITHUB_OUTPUT
-          log=$(git log origin/${{ inputs.to_branch }}..origin/${{ inputs.ref }} --pretty=format:'%h - %s (%an)')
+          log=$(git log origin/${{ inputs.to_branch }}..${{ inputs.ref }} --pretty=format:'%h - %s (%an)')
```

Comment:
I assume this is fine as long as branch names  are different, which should lways be the case.